### DISCUSSION TITLE
Variable Naming Conventions

### DISCUSSION THREAD

**Discussion by spkelle2** [ambros-gleixner/MIPcc23#21] [Q&A] [answered]
I've been using CBC/CLP for my submission, and it appears the ability for CoinUtils to read in column names from .mps files (`OsiClpSolverInterface::readMps(const char *filename, bool keepNames, bool allowErrors)`) is broken. The result is that my columns are renamed to and saved as `C<column index>`. Coincidentally, I've noticed that in the provided test set, at least for the `.mps` files I've manually checked myself, the columns are named following the same convention.

This being the case, can I expect columns in the instances used for the competition to all be named by the same convention or will it be acceptable for me to save columns in that format under the assumption that I save all columns and they share indices with the originally assigned column names?

**Comment by spkelle2** [comment, marked as answer]
Turns out this isn't an issue. So please feel free to disregard. For anyone with a similar issue, just make sure to provide a 0 for the second argument in `OsiClpSolverInterface::getColName(int colIndex, unsigned maxLen=static_cast< unsigned >(std::string::npos))` and to cache the names before your solve since CBC does not persist them.

**Comment by ambros-gleixner** [comment]
Happy you could solve it!
